config/branding/config.ini

```
@@ -49,6 +49,8 @@ admin = "hpc@it.umass.edu"
 admin_name = "Unity Cluster Admins"
 sender = "updates@unity.rc.umass.edu"
 sender_name = "Unity Cluster Updates"
+pi_approve_email = "piapproval@umass.edu"
```

```suggestion
pi_approve = "piapproval@umass.edu"
```

To keep it consistent with the others